### PR TITLE
Update Google AppEngine releases to 1.9.30

### PR DESCRIPTION
**Pull request by toonetown**
(updates the go-32, java, and python platforms.  The go-64 platform is already at version 1.9.30)

### CODE REVIEW

**Comment by bfontaine** [comment]
Would you mind making one commit per formula? Thanks!

File: Library/Formula/app-engine-python.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class AppEnginePython < Formula
   desc "Google App Engine"
   homepage "https://cloud.google.com/appengine/docs"
-  url "https://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.27.zip"
-  sha256 "6192f295969dabf8659ce9a698450154f7c8c35b89c6e3cb52908c8f50d7c1f4"
+  url "https://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.30.zip"
+  sha256 "d908f031672f8a95da0cc9230253a9985cfd57f6721d89606ccea21c8e00b471"
   revision 1
```
Comment (DomT4) [inline Library/Formula/app-engine-python.rb]: Please remove `revision` lines outside of bottle blocks on stable updates, Thanks!

Comment (toonetown) [inline Library/Formula/app-engine-python.rb]: Sorry - I didn't know about the revision lines

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/2402232def3c5a6760da45457aa707e63bd83ad8, https://github.com/Homebrew/homebrew/commit/57bc0feb9d1215e225876bec68b5f595594a70a0 & https://github.com/Homebrew/homebrew/commit/923fc28eab6b78295c8ae45dbba3749372699b26.

Just a nit but for version updates the preferred commit message style is:
```
app-engine-python 1.9.30
```
Or similar.

Thank you for your further contributions to Homebrew @toonetown! :beer:

**Pull request closed without merging**